core/src/main/scala/kafka/admin/ConfigCommand.scala

```diff
@@ -576,6 +594,8 @@ object ConfigCommand extends Logging {
         if (entityName.nonEmpty)
           validateBrokerId()
         (ConfigResource.Type.BROKER_LOGGER, None)
+      case ConfigType.ClientMetrics =>
+        (ConfigResource.Type.CLIENT_METRICS, Some(ConfigEntry.ConfigSource.DYNAMIC_CLIENT_METRICS_CONFIG))
```

Comment:
Probably ought to have a non-empty entity name, or describeAll.

Reply:
Yeah, I need to support describe all which shall require either adding an API in admin client or need to work something with in-memory cache of subscriptions. I ll figure that out as we proceed with other code changes.